homeworks/becseipeter/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,43 @@
+all_round = int(input("How many round do you wanna play?\n"))
```

Comment:
Ezt a bekérést, és az 5. soron lévő validációt egybe lehetne olvasztani, és akkor a kódban csak egyszer kérnél be inputot:

```
while True:
    all_round = int(input("How many rounds do you want to play? "))
    if all_round % 2 != 0:
        break
    print("The number of rounds must be odd. Try again.")
```